bin/node/runtime/src/lib.rs

```diff
@@ -468,11 +468,102 @@ parameter_types! {
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 256;
-}
+	// enable the below flag to allow dynamic damping of validator count per era.
+	pub const EnableAutomaticValidatorUpdatePerEra: bool = true;
```

Comment:
Do we need enable? I recommend assuming that this is always enabled, and if you want a noop, you just use `sp_runtime::traits::Identity` instead of `AutomaticValidatorUpdatePerEra `.

Reply:
Noted 👍